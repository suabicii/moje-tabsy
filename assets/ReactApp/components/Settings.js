import React from "react";

function Settings() {
    return (
        <>
            <h1 className="text-center">Ustawienia</h1>
        </>
    );
}

export default Settings;