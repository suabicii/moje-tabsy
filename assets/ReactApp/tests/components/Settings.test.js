import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import Settings from "../../components/Settings";

it('should correctly render Settings Page', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Settings/>);

    expect(renderer.getRenderOutput()).toMatchSnapshot();
});