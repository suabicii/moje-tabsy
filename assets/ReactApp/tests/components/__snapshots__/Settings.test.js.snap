// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Settings Page 1`] = `
<React.Fragment>
  <h1
    className="text-center mt-5 mt-md-0"
  >
    Ustawienia
  </h1>
  <SettingsForm />
</React.Fragment>
`;
